podcast/2.12.m

```diff
@@ -0,0 +1,933 @@
+
+**Beyang Liu:**
+
+Cool. It sounds like you're a bit of a troublemaker at first–the insult generator and then the satirical newspaper/kind of fake email. When did that start? Was that just your personality as a kid always or was there some kind of catalyst or trigger there?
+
+**Joseph Nelson:**
+
+Honestly, this makes me sound a lot cooler than I was in school. And the reason I was able to kind of get away with this stuff is because I was the least suspicious kid. I always got my homework in on time and always focused on doing well and in classes, I kind of kept to myself and–we can just come out and say it–I was one of the nerds at school. But I had this curious side to me of like, I don't know, why not try some of this stuff? So in school, I wasn't like the class troublemaker or something like that. In fact, I was just much more kind of like that quiet kid, who knows what he's up to and working on, which I think probably made it so I could kind of get away with some of this stuff.
+
+**Beyang Liu:**
+
+Yeah. That's awesome. So in college, despite the success that you had with the calculator program, you didn't go into studying computer science as a major, is that right? You studied economics instead?
+
+**Joseph Nelson:**
+
+That's correct, yeah. So my degree is in economics.
+
+**Beyang Liu:**
+
+Cool. What was that like?
+
+**Joseph Nelson:**
+
+I liked it quite a bit. I mean the bad joke I kind of make is that economics is kind of like a gradient descent into machine learning land. Economics is a study that basically takes a representation of the world and then oversimplifies it and uses equations to try to optimize that oversimplification, especially microeconomics, like rent. 
+
+Take rent, for example. If you build more housing, despite what they think in San Francisco, prices will generally go down. And we could probably build an equation that roughly approximates that if we build 100 more units, then we might expect rent to drop even a little bit and that's going to be a very brutal, imperfect representation, but it's directionally correct. And then we can take that formula and optimize it. So, pretty soon, you start to get into econometrics, and I realized it was like, okay, this is all just applied statistics. That’s all this economics stuff is.
+
+And then the deeper I got into the applied statistics side of things, the more I realized that programming is where the action happens. It's like, what model are you using? How can you make sure that things run efficiently? And I guess keeping with my streak now of getting in trouble. I turned in my thesis in Python instead of Stata, which apparently you weren't supposed to do. The argument was it was less reproducible and I was like, are you kidding me? No one has to pay to reproduce my code–here's everything that you need to reproduce my senior thesis and all that sort of stuff. I took a couple of CS classes here and there, but yeah, my degree is economics and lots of stats.
```

Comment:
```suggestion
And then the deeper I got into the applied statistics side of things, the more I realized that programming is where the action happens. It's like, what model are you using? How can you make sure that things run efficiently? And I guess keeping with my streak now of getting in trouble, I turned in my thesis in Python instead of Stata, which apparently you weren't supposed to do. The argument was it was less reproducible and I was like, are you kidding me? No one has to pay to reproduce my code–here's everything that you need to reproduce my senior thesis and all that sort of stuff. I took a couple of CS classes here and there, but yeah, my degree is economics and lots of stats.
```